go/interpreter/lfvm/instructions_test.go

```diff
@@ -282,15 +282,13 @@ func TestCreateChecksBalance(t *testing.T) {
 	}
 }
 
-func TestBlobHash(t *testing.T) {
-
+func TestBlobHash_PushesCorrectValueOnStack(t *testing.T) {
```

Comment:
gas and revision can be removed from the test case parameters. 
`stack.push(uint256.NewInt(0))` is also used in all cases so it can be done as part of the test instead of the test cases.

Reply:
I removed gas and revision but not all cases push 0, the last case is pushing 1.